### PR TITLE
[modular] fixes some lewd furnitures spawnable inside people's hands

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

it's possible for some lewd furniture to spawn in people's hands and they will delete themselves if they try to interact with it

## How This Contributes To The Skyrat Roleplay Experience

it's a fix

## Changelog

:cl:
fix: fixed lewd furnitures spawning in people's hands
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_machinery/milking_machine.dm [hunk 1]
```diff
@@ -1043,7 +1042,7 @@
 		if(get_turf(user) == get_turf(src))
 			return
 		else if(I.use_tool(src, user, 8 SECONDS, volume=50))
-			var/obj/structure/chair/milking_machine/N = new M(src.loc)
+			var/obj/structure/chair/milking_machine/N = new(get_turf(user))
 			if(istype(src, /obj/item/milking_machine/constructionkit))
 				if(current_color == "pink")
 					N.machine_color = N.machine_color_list[1]
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_machinery/milking_machine.dm]: While you're there, replace `N` by something more significant, please.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/dancing_pole.dm [hunk 2]
```diff
@@ -176,8 +175,7 @@
 		to_chat(user, span_notice("You begin unfastening the frame from the floor and ceiling..."))
 		if(P.use_tool(src, user, 8 SECONDS, volume=50))
 			to_chat(user, span_notice("You disassemble the stripper pole."))
-			var/obj/item/polepack/C = new
-			C.loc = loc
+			new /obj/item/polepack(get_turf(user))
```
Comment (jjpark-kb) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/dancing_pole.dm:178]:
```suggestion
			new /obj/item/polepack(get_turf(src))
```
perhaps this instead? I don't know, this is just preference at this point, ahaha.

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/bdsm_furniture.dm [hunk 3]
```diff
@@ -288,8 +287,7 @@
 		to_chat(user, span_notice("You begin unfastening the frame of x-stand..."))
 		if(P.use_tool(src, user, 8 SECONDS, volume=50))
 			to_chat(user, span_notice("You disassemble the x-stand."))
-			var/obj/item/x_stand_kit/C = new
-			C.loc = loc
+			new /obj/item/x_stand_kit(get_turf(user))
```
Comment (jjpark-kb) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/bdsm_furniture.dm:290]:
```suggestion
			new /obj/item/x_stand_kit(get_turf(src))
```
perhaps this instead? I don't know, this is just preference at this point, ahaha.

**Comment by tf-4** [comment]
Address reviews or accept your fate

**Comment by jupyterkat** [comment]
preference reviews are irrelevant

**Review by tf-4 — APPROVED** [review]

**Pull request merged**